HospitalSystem.Core/People/Patient/Patient.cs

```diff
@@ -6,12 +6,21 @@ public class Patient : Person
         [BsonElement]
         public MedicalRecord MedicalRecord {get; set;}
         public List<CheckupChangeLog> CheckupChangeLogs {get; set;}
-        public Patient(string firstName, string lastName, MedicalRecord medicalRecord) : base(firstName, lastName)
+        public TimeSpan WhenToRemind {get; set;}
+        public Patient(string firstName, string lastName, MedicalRecord medicalRecord, TimeSpan? whenToRemind = null) : base(firstName, lastName)
         {
             FirstName = firstName;
             LastName = lastName;
             MedicalRecord = medicalRecord;
             CheckupChangeLogs = new List<CheckupChangeLog>();
+
+            if (whenToRemind is null){
+                WhenToRemind = TimeSpan.FromHours(1);
+            }
+            else{
+                WhenToRemind = (TimeSpan)whenToRemind;
+            }
```

Comment:
Why do you allow null passing? When does that happen?

Reply:
This is just a way to make default value to WhenToRemind, i would do it in parameters but it creates an error